<launch>
   <include file="$(find pr2_overhead_grasping)/launch/simple_grasp_pipeline.launch"/>
    <!--<node pkg="pr2_overhead_grasping" name="overhead_grasping" type="overhead_grasping.py" args="-s" output="screen"/>-->

  <node pkg="pr2_collision_monitor" type="collision_monitor" name="l_collision_monitor" output="screen" >
    <param name="arm" type="str" value="l"/>
    <param name="training_mode" type="bool" value="true"/>
    <param name="behavior_name" type="str" value="overhead_grasp"/>
    <param name="data_filename" type="str" value="$(find pr2_overhead_grasping)/data/overhead_grasping_data.bag" />

  </node>                 

</launch>
